### ISSUE TITLE
Would it be possible to port this feature to NextCloud 13 and 14?

### ISSUE THREAD

**Issue by alauzon** [Stibila/defaultgroup#5]
I would love to use it on my NextCloud v13.

**Comment by simonbuehler** [comment]
yes, as we authenticate against imap its not possible to set default groups, this would help so much!

**Comment by Stibila** [comment]
For IMAP authentication, there is workaround using login hook. If you enable it in settings, it will add user to default group(s) whenever they log-in. This app was originally created to use with IMAP authentication.

As for NC 13, some reported, that it works. For 14, you can test it and report back. I no longer use NC, (this is why I gave this project to @bodangren), so I can't help more.

**Comment by simonbuehler** [comment]
ok just changed max version to 14 and it does work, thanks!

**Comment by bodangren** [comment]
I'll make the change. Thanks for the report.

**Comment by bodangren** [comment]
> I would love to use it on my NextCloud v13.

Has worked on 13 since 2018-02-28, sir.
